e2etests/vuln_test.go

```diff
@@ -6,10 +6,10 @@ package e2etests
 import (
 	"context"
 	"fmt"
+	"slices"
```

Comment:
This change is not related to migration

Reply:
@janisz Here is PR to pull that out: https://github.com/stackrox/scanner/pull/1572

Reply:
Changes for `slices` are removed.